### PR TITLE
net/ntopng: Add community mode checkbox

### LINKED ISSUE #2185: Feature request: ntopng plugin - allow additional settings for ntopng.conf
The configuration form for ntopng plugin doesn't allow any custom/additional configuration settings as described at https://www.ntop.org/guides/ntopng/how_to_start/configuration_file.html

**ntopng.conf** gets rewritten each time the form is saved, leaving no place to add custom configuration settings to ntopng.

Suggestion: an additional field 'Custom options' that would be appended at the bottom of generated **ntopng.conf** - similar to Custom options field on Unbound configuration form.

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #2185

### CODE REVIEW

File: net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
     <description>ntopng configuration</description>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
     <items>
         <enabled type="BooleanField">
             <default>0</default>
             <Required>Y</Required>
         </enabled>
+        <community type="BooleanField">
+            <default>0</default>
```
Comment (fichtner) [inline net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml:11]: Why not default to enabled? I even thought about using it unconditionally?

Comment (mimugmail) [reply net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml:11]: Becasue for me it also works without it, why changing and may breaking running setups.

Comment (mihakralj) [reply net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml:11]: I agree with mimugmail.
By default ntopng 4.3 starts in Enterprise edition for 10 minutes and then downscales to community edition. Forcing it to community edition with --community makes the interface consistent for non-licensed clients.
Clients that want to try Enterprise features - or even purchase it - should have an option to run it without this switch.

**Review by mihakralj — CHANGES_REQUESTED** [review]
Help text:
This will force ntopng to start in Community version. If left unchecked, ntopng will demonstrate enterprise features for the first 10 minutes. For more information on ntopng versions and licensing: https://www.ntop.org/products/traffic-analysis/ntop/

File: net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml (hunk 1 above)
Comment (fichtner) [reply net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml:11]: It doesn’t really answer the need for sane default question but we can take it to the other PR.

**Comment by fichtner** [comment]
Waiting for #2292 to reappear.

**Pull request closed without merging**